grow/translators/google_sheets.py

```diff
@@ -396,10 +425,81 @@ def _generate_style_requests(self, sheet_id):
                 },
             },
         })
+
+        # Highlight missing translations.
+        requests.append({
+            'addConditionalFormatRule': {
+                'rule': {
+                    'ranges': [{
+                        'sheetId': sheet_id,
+                        'startColumnIndex': 1,
+                        'endColumnIndex': 2,
+                        'startRowIndex': self.HEADER_ROW_COUNT,
+                    }],
+                    'booleanRule': {
+                        'condition': {'type': 'BLANK'},
+                        'format': formats['missing_cell']
+                    }
+                },
+                'index': 0
+            }
+        })
+
+        # Protect the original values.
+        requests += self._generate_style_protected_requests(sheet_id, sheet, {
+            'protectedRangeId': sheet_id + 1000001,  # Keep it predictble.
+            'range': {
+                'sheetId': sheet_id,
+                'startColumnIndex': 0,
+                'endColumnIndex': 1,
+                'startRowIndex': self.HEADER_ROW_COUNT,
+            },
+            'description': 'Original strings can only be edited in the source files.',
+            'warningOnly': True,
+        })
+
+        # Protect the location values.
+        requests += self._generate_style_protected_requests(sheet_id, sheet, {
+            'protectedRangeId': sheet_id + 1000002,  # Keep it predictble.
+            'range': {
+                'sheetId': sheet_id,
+                'startColumnIndex': 2,
+                'endColumnIndex': 3,
+                'startRowIndex': self.HEADER_ROW_COUNT,
+            },
+            'description': 'Description strings can only be edited in the source files.',
```

Comment:
Should be something more like "Source strings can only be edited in source code." -> since the developer is the person who uploads this (or CI uploads this), and non-technical translators will be the ones seeing these settings, most likely.